comitup/config.py

```diff
@@ -15,12 +15,23 @@
 import os
 import random
 import shutil
+import re
+import logging
+import subprocess
 
 from comitup import persist
 
+log = logging.getLogger('comitup')
+
 PERSIST_PATH = "/var/lib/comitup/comitup.json"
 CONF_PATH = "/etc/comitup.conf"
 BOOT_CONF_PATH = "/boot/comitup.conf"
+DEFAULT_AP = "comitup-<nnn>"
+
+# ap_name regex patterns
+REGEX_APNAME_FORMAT = r'[a-zA-Z\<\>](\w*)-\<([nMs]+?)\>'  # validate format name-<###>|<hostname>-<###>
```

Comment:
This restriction is unnecessary. SSIDs can be more flexible than this, and there is no reason for Comitup to be more restrictive.
Validation on the interpolated string could be useful - e.g. length between 2 and 32 characters, and a check on the characters. Probably not worth the effort.

Reply:
I did improve the regex to look for the correct length for each type of format string.
I only intended to validate the ap name from commit.conf... 
I agree, didn't intend on restricting valid SSIDs...